service/controller/resource/workermigration/testdata/azuremachinepool.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureMachinePool
+metadata:
+  labels:
+    azure-operator.giantswarm.io/version: 4.2.0
+    cluster.x-k8s.io/cluster-name: c6fme
+    giantswarm.io/cluster: c6fme
+    giantswarm.io/machine-pool: c6fme
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 12.0.0
+  name: c6fme
+  namespace: default
+spec:
+  location: westeurope
+  template:
```

Comment:
I believe we need the `datadisks`.

Reply:
Theoretically yeah, but since this isn't used for ignition rendering, it's not strictly necessary for testing purposes.